adapters/connext/src/index.ts

```diff
@@ -8,7 +9,6 @@ const input = {
 const fileName = 'output.csv';
 console.log('Getting TVL at block:', input.blockNumber);
 
-
 // returns all user balances at the input block by looking at the latest
 // balance for each user and token on the subgraph, capped at given block.
 getUserTVLByBlock(input).then((data) => {
```

Comment:
@LayneHaber can we just make this function to return the list of csvrow.

Reply:
the `getUserTVLByBlock` function does just return a `OutputDataSchemaRow[]` type.

updated the file to only export this helper instead of running it